fluent-bundle/src/types/number.rs

```diff
@@ -8,6 +8,28 @@ use intl_pluralrules::operands::PluralOperands;
 use crate::args::FluentArgs;
 use crate::types::FluentValue;
 
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
+pub enum FluentNumberKind {
```

Comment:
Suggestion: Please add docs here, with a link to the relevant Fluent documentation, and an example of what the `.ftl` syntax looks like.